x-pack/filebeat/module/coredns/log/test/coredns-json.log-expected.json

```diff
@@ -54,9 +65,20 @@
             "RA"
         ],
         "coredns.response.size": 83,
+        "dns.header_flags": [
+            "QR",
+            "RD",
+            "RA"
+        ],
+        "dns.id": "6966",
+        "dns.question.class": "IN",
+        "dns.question.name": "httpbin.org.",
```

Comment:
Could you remove the trailing dot from domain lookups?

Reply:
Good call.